components/eamxx/cime_config/namelist_defaults_scream.xml

```diff
@@ -490,18 +499,37 @@ be lost if SCREAM_HACK_XML is not enabled.
     <column_conservation_checks_fail_handling_type>Warning</column_conservation_checks_fail_handling_type>
     <check_all_computed_fields_for_nans type="logical">true</check_all_computed_fields_for_nans >
     <property_check_data_fields type="array(string)" doc="list of additional data fields to output in property checks (only for physics grid)">phis,landfrac</property_check_data_fields>
+    <enable_intensive_observation_period type="logical" doc="Enable intensive observation period. Currently the only use case is DP-EAMxx">false</enable_intensive_observation_period>
+    <enable_intensive_observation_period COMPSET=".*DP-EAMxx">true</enable_intensive_observation_period>
   </driver_options>
 
   <!-- E3SM Simulation Settings -->
   <e3sm_parameters>
     <e3sm_casename>${CASE}</e3sm_casename>
   </e3sm_parameters>
 
+  <!-- Settings for IOP -->
+  <intensive_observation_period_options>
```

Comment:
Can you add doc strings for all these params too? You should be able then to verify the docs is correct by looking at [this](https://e3sm-project.github.io/scream/pr-preview/pr-2739/common/eamxx_params/) auto-generated page.